shuup/front/apps/registration/templates/shuup/registration/register.jinja

```diff
@@ -18,7 +18,7 @@
                     {% csrf_token %}
                     {% for field in form.hidden_fields() %}{{ field|safe }}{% endfor %}
                     {% for field in form.visible_fields() %}{{ render_field(field) }}{% endfor %}
-                    {% if get_shop_configuration("allow_company_registration") %}
+                    {% if shuup.general.is_company_registration_allowed() %}
```

Comment:
missing the request param

Reply:
The request is optional for the thing. Just what happened there with customer information was that for some reason the context didn't have the request. Since we pass request for that macro already I thought there might be a reason for all this and added option to pass the request manually from the places where the context is weird.